File: C++ Assignments/Assignment 2/leap year.cpp
#include<iostream>
using namespace std;
class Leapyear
{
int year;
public: void check() 
{
cout<<"Enter year \n";
cin>>year;
if(((year%4 == 0) && (year%100 != 0)) || (year%400 == 0)) 
{
cout<<"Leap year\n";
}
else
{
cout<<"Not leap year\n";
}
}
};

int main()
{   
Leapyear obj1;
obj1.check();
return 0;
}

File: JavaAssignment-2/LeapYear.java
import java.util.*;
class LeapYear
{
public static void main(String[] args)
{
int year;
System.out.println("Enter any year");
Scanner sc=new Scanner(System.in);
year=sc.nextInt();
if(((year%4==0)&&(year%100!=0))||(year%400==0))
System.out.println("Leap Year");
else
System.out.println("Not a leap year");
}
}